test/src/e2e_vm_tests/test_programs/should_pass/language/contract_caller_as_type/.gitignore

```diff
@@ -0,0 +1,2 @@
+out
```

Comment:
That's not needed I assume.

Reply:
Doesn't really hurt to keep these gitignores?

Reply:
![image](https://user-images.githubusercontent.com/59666792/162346205-a7a10670-ee04-4f18-a845-c7227ef84d66.png)